solr/modules/ltr/src/test/org/apache/solr/ltr/feature/TestFieldValueFeature.java

```diff
@@ -250,6 +250,31 @@ public void testIfADocumentDoesntHaveAFieldDefaultValueIsReturned() throws Excep
     assertJQ("/query" + query.toQueryString(), "/response/numFound/==1");
     assertJQ("/query" + query.toQueryString(), "/response/docs/[0]/id=='42'");
 
+    final String docs0fv_dense_csv =
```

Comment:
Are all these tests just refactoring? It's hard to review, and if there's no actual test change, I would strongly encourage leaving them as they were.
When bug-fixing, the less we touch the tests, the better, so it's clear nothing has been broken by the bugfix,